backend/src/Core/FurnitureShop.Core.Contracts/Mobile/Products/ProductsDTOs.cs

```diff
@@ -3,30 +3,42 @@
 
 namespace FurnitureShop.Core.Contracts.Mobile.Products
 {
-    public class ProductDetailsDTO : ProductInfoDTO
-    {
-        public string Description { get; set; }
-        public string? ModelUrl { get; set; }
-    }
-    public class ProductInfoDTO
+    public class ProductDTOBase
     {
         public string Name { get; set; }
         public double Price { get; set; }
         public double? AverageRating { get; set; }
         public string? PreviewPhotoURL { get; set; }
-        public bool InFavourites { get; set; }
-        public bool InShoppingCart { get; set; }
         public Guid? CategoryId { get; set; }
     }
+    public class ProducDetailsDTOBase : ProductDTOBase
+    {
+        public string Description { get; set; }
+        public string? ModelUrl { get; set; }
+    }
 
-    public class ProductWithDetailsDTO
+    public class ProductDetailsDTO : ProducDetailsDTOBase
     {
         public Guid Id { get; set; }
-        public ProductDetailsDTO ProductDetails { get; set; }
     }
-    public class ProductDTO
+    public class ProductDTO : ProductDTOBase
     {
         public Guid Id { get; set; }
-        public ProductInfoDTO ProductInfo { get; set; }
+    }
+    public class ProductUSerViewDTO : ProductDTO
```

Comment:
Type User*

Reply:
Lets have these objects not shared between mobile and web, just create ProductDTO in Mobile namespace

Reply:
Still unresolved, remove ProductUSerViewDTO class and make ProductDTO in Mobile and Web namespaces separately

Reply:
sorki źle zrozumiałem komentarz